frontend/internal-packages/schema-bench/src/cli/executeOpenaiUnified.ts

```diff
@@ -19,17 +19,42 @@ import {
 
 type DatasetResult = { datasetName: string; success: number; failure: number }
 
+// Outputs: write latest and optional archive ID for correlation
+const RUN_ID = new Date().toISOString().replace(/[:.]/g, '-')
+
 const InputSchema = v.object({
   input: v.string(),
 })
 
 async function executeCase(
   executor: OpenAIExecutor,
+  datasetName: string,
   datasetPath: string,
   caseId: string,
   input: OpenAIExecutorInput,
 ): Promise<Result<void, Error>> {
-  const result = await executor.execute(input)
+  const threadId = [datasetName, caseId, RUN_ID].join(':')
+  /*
+   * Retain this server-only log to deterministically correlate runs with LangSmith (thread_id/runId), surface an optional trace search URL, and aid troubleshooting with no secrets/PII and minimal overhead even if tracing is delayed or unavailable.
+   */
+  // biome-ignore lint/suspicious/noConsole: Allow server-side console output for LangSmith trace correlation
+  console.log(
+    `[schema-bench] executing case: dataset=${datasetName} case=${caseId} thread_id=${threadId}`,
+  )
```

Comment:
If it's necessary output, please use `console.info`. This will not be flagged by biome.

```suggestion
  console.info(
    `[schema-bench] executing case: dataset=${datasetName} case=${caseId} thread_id=${threadId}`,
  )
```

Reply:
[♻️(schema-bench): Refactor CLI scripts to use tracing utilities and i…](https://github.com/liam-hq/liam/pull/3643/commits/b66e786cf84217a0799a584776f4e13c88c5b396)

I forgot about console.info🙏 I've taken care of it here!